dask/dataframe/tests/test_utils_dataframe.py

```diff
@@ -126,13 +126,19 @@ def __next__(self):
     assert len(meta.index) == 0
 
     # Categoricals
-    meta = make_meta({"a": "category"}, parent_meta=df)
+    meta = make_meta({"a": "category"}, parent_meta=df)  # fails
     assert len(meta.a.cat.categories) == 1
     assert meta.a.cat.categories[0] == UNKNOWN_CATEGORIES
     meta = make_meta(("a", "category"), parent_meta=df)
     assert len(meta.cat.categories) == 1
     assert meta.cat.categories[0] == UNKNOWN_CATEGORIES
 
+    # Categorials with Index
+    meta = make_meta({"a": "category", "b": "int64"}, index=idx)
+    assert len(meta.a.cat.categories) == 1
+    assert meta.index.dtype == "int64"
+    assert len(meta.index) == 0
```

Comment:
I think `Index` objects have an `.empty` attribute for these types of situations 

```suggestion
    assert not meta.index.empty
```